pep-0620.rst

```diff
@@ -27,10 +27,10 @@ changes are already completed. The `Process to reduce the number of
 broken C extensions`_ dictates the rhythm.
 
 
-PEP rejected
-============
+PEP withdrawn
+=============
 
-PEP rejected by its author since the scope is too broad and the work is
+This PEP was withdrawn by its author since the scope is too broad and the work is
 distributed on multiple Python versions which makes it difficult to take
 a decision on the overall PEP. It was splitted in new PEPs with a more
 narrow and better defined scope like :pep:`670`.
```

Comment:
```suggestion
distributed over multiple Python versions which makes it difficult to make
a decision on the overall PEP. It was split in new PEPs with a
narrower and better defined scope, like :pep:`670`.
```

Reply:
Also, if this was split into multiple PEPs, can we list them all here? In addition to 670?

Reply:
```suggestion
distributed over multiple Python versions, which makes it difficult to make
a decision on the overall PEP. It was split into new PEPs with
narrower and better defined scopes, like :pep:`670`.
```

The above, with some additional grammar fixes.

Just as a sidenote—"take a decision" is unknown in AmE, but apparently valid phrasing in at least some variations of BrE and is apparently used in many dialects of European English (particularly so as the _lingua franca_ of international intra-regional cooperation and decisionmaking), with a specific and subtly different meaning from the more common "make a decision" used most of the time in BrE and all of the time in AmE. As I understand, in that context "take a decision" refers to the specific moment of making a formal, official, procedural decision making, whereas "make a decision" refers to both that and the overall decision-making process, and is used for more informal/personal decisions.